.github/workflows/ci_reusable_wf.yml

```diff
@@ -321,7 +322,7 @@ jobs:
         env:
           RUSTFLAGS: "-D warnings"
         # compilations options must be kept in sync with cargo-test
-        run: cargo build --all-features
+        run: cargo build ${{ env.COMMON_CARGO_FLAG_TESTS }}
```

Comment:
typo
```suggestion
        run: cargo build ${{ env.COMMON_CARGO_TEST_FLAGS }}
```

Reply:
so github does not complain if I refer to something that is not in `env`?

Reply:
i guess it returns an empty string if the env can't be found.